commcare_connect/templates/opportunity/user_visit_verification.html

```diff
@@ -0,0 +1,476 @@
+{% extends 'tailwind/base.html' %}
+
+{% load static %}
+{% load django_tables2 %}
+{% load i18n %}
+
+{% block css %}
+  {{ block.super }}
+  <link rel="stylesheet" href="{% static 'bundles/css/mapbox.css' %}">
+{% endblock %}
+
+{% block javascript %}
+  {{ block.super }}
+  <script src="{% static 'bundles/js/mapbox-bundle.js' %}" defer></script>
+{% endblock javascript %}
+
+{% block content %}
+  <ul class="breadcrumb">
+      {% if opportunity.managed %}
+      <li>
+          <!-- Todo; update to program:home -->
+          <a href='{% url "program:list" request.org.slug %}'>Programs</a>
+          <i class="fa-solid fa-chevron-right"></i>
+      </li>
+      {% endif %}
+      <li>
+          <a href='{% url "opportunity:list" request.org.slug %}'>Opportunities</a>
+          <i class="fa-solid fa-chevron-right"></i>
+      </li>
+      <li>
+          <a href='{% url "opportunity:detail" request.org.slug opportunity.pk %}'>{{ opportunity.name }}</a>
+          <i class="fa-solid fa-chevron-right"></i>
+      </li>
+      <li>
+          <a href='{% url "opportunity:user_visits_list" request.org.slug opportunity.pk opportunity_access.pk %}'>Worker</a>
+      </li>
+  </ul>
+  <div class="w-full space-y-4">
```

Comment:
can you explain a bit about some of this deeply nested CSS? What is it doing, and is this something we will need to be able to replicate elsewhere?